compiler/packages/babel-plugin-react-compiler/src/Utils/Result.ts

```diff
@@ -90,26 +90,29 @@ export function Ok<T>(val: T): OkImpl<T> {
 }
 
 class OkImpl<T> implements Result<T, never> {
-  constructor(private val: T) {}
+  #val: T;
```

Comment:
I think this is the only place we use this syntax and rollup didn't like it, and it was easier to switch to private identifiers than deal with more babel plugins